gql/filter.go

```diff
@@ -132,6 +141,7 @@ func parseASTValue(in interface{}) (interface{}, error) {
 type listFilter struct {
 	fieldName string
 	op        Comparator
+	json      *listFilterJSON
```

Comment:
What you have works but just to clarify a bit on what I was thinking here you would add operationName string, Argument string here.